templates/data-store-pv.yaml

```diff
@@ -20,7 +20,7 @@
 # the volume is actually used by other pods as a different mode
 # there is quite a good explanation here: https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/100
 # as to the slightly anomalous behaviour
-
+{{- if .Values.global.persistence.statefulSet.pv.enabled }}
```

Comment:
These are not part of a statefulset, therefore this isn't  required here

Reply:
Good one.  Rolled it back, double checked by running though the testing and all is well.